### PR TITLE
[FEATURE] Use PSR-20 time and attach time to projectNode

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: tests/Integration/IntegrationTest.php [hunk 1]
```diff
@@ -64,9 +66,13 @@ public function testHtmlIntegration(
             system('mkdir ' . escapeshellarg($outputPath));
 
             $this->prepareContainer($configurationFile);
+
             $command = $this->getContainer()->get(Run::class);
+
             assert($command instanceof Run);
 
+            $command->setClock(new MockClock('2023-01-01 12:00:00'));
```
Comment (jaapio) [inline tests/Integration/IntegrationTest.php:74]: You can set this in the `\phpDocumentor\Guides\DependencyInjection\TestExtension` then there is no reason the have the setClock method. You can register the MockClock there as a service with a certain argument.

File: tests/Integration/IntegrationTest.php [hunk 2]
```diff
@@ -39,6 +40,7 @@ class IntegrationTest extends ApplicationTestCase
     protected function setUp(): void
     {
         setlocale(LC_ALL, 'en_US.utf8');
+        // todo: mock time
```
Comment (jaapio) [inline tests/Integration/IntegrationTest.php:43]: This seems to be obsolete?

File: tests/Integration/IntegrationTest.php (hunk 1 above)
Comment (linawolf) [reply tests/Integration/IntegrationTest.php:74]: How could I set the mock time there?

Comment (linawolf) [reply tests/Integration/IntegrationTest.php:73]: Ok, figured it out, setClock is removed

**Comment by linawolf** [comment]
@jaapio Please have another look

**Review by jaapio — APPROVED** [review]

**Pull request merged**